### PR TITLE
basic django 1.8 compatibility

### PR DESCRIPTION
**Pull request by imposeren**
this should also solve question raised in #84 . But maybe readme note about template engine is also required

### CODE REVIEW

**Comment by imposeren** [comment]
this also affects #85

File: docs/quick-start.rst [hunk 1]
```diff
@@ -34,6 +34,32 @@ context processors are included in this list. For a list of default
...
+    TEMPLATES = [ # example config untill 'context_processors' your config maydiffer
+        {
+            'BACKEND': 'django.template.backends.django.DjangoTemplates',
+            'DIRS': [
+            ],
+            'APP_DIRS': True,
+            'OPTIONS': {
+                'context_processors': [
+                	# add required context processors here:
+                    'django.core.context_processors.request',
```
Comment (mlavin) [inline docs/quick-start.rst]: This has been moved to `django.template.context_processors.request` and this old location is set to be removed in 1.10.

**Comment by mlavin** [comment]
One very minor comment. Otherwise this is amazing. Thank you! :hibiscus: :heart_eyes_cat:

**Comment by imposeren** [comment]
> One very minor comment. Otherwise this is amazing. Thank you! 

I've fixed path to context processor. I may rebase this all into one commit if you'd like it so.

**Comment by mlavin** [comment]
I don't mind all of the commits in the history if you don't. I think this can be merged without the rebase.

**Comment by imposeren** [comment]
Ok.

**Pull request merged**